force-app/main/default/lwc/psElevateTokenHandler/psElevateTokenHandler.js

```diff
@@ -105,7 +105,7 @@ class psElevateTokenHandler {
                 if (typeof event.data === 'object') {
                     component.handleMessage(event.data);
                 } else {
-                    const message = JSON.parse(event.data);
+                    const message = event.data;
```

Comment:
I'm trying to follow the logic, does this change require an update to `handleMessage` in `geFormWidgetTokenizeCard.js`? That's one of the `component`s that get passed into this.

Reply:
That is a good point, I'm going to look into the logic that gets called there and see any changes that need to be made, the original issue appeared to have to do with how the `action` value in `event` was being unnecessarily stringified so when you tried to reference it, it was returning as `undefined` but I will look into the `geFormWidgetTokenizeCard` side as well.

Reply:
@lparrott So I took a loot at the `geFormWidgetTokenizeCard.js` logic, it looks like the `message` value that gets passed through there is already stringified and then gets passed through here and parsed. I would feel a bit more comfort going with this solution though if I can have ways to simulate errors-based events just to ensure those bodies are being parsed correctly. Would manually passing an error as an event be adequate in this case?

Reply:
Oh definitely, if you can test that and it comes through fine I think we're in good shape!

Reply:
@lparrott So I went ahead and manually passed in an error to test that it was parsing the error properly and it seemed to properly handle that. It would be a bit more comforting to be able to QA this in an error state as well but not sure how feasible that is.  I also tested ACH as an option and it appears to load fine: 
<img width="1475" alt="Screen Shot 2022-11-04 at 10 47 11 AM" src="https://user-images.githubusercontent.com/105941702/200003955-6bee97e9-a066-411c-ac3e-9afe5a18fb59.png">
I did notice when I selected `Paypal` this showed up:
<img width="1481" alt="Screen Shot 2022-11-04 at 10 49 00 AM" src="https://user-images.githubusercontent.com/105941702/200004561-3505fab8-01b6-4456-95a1-fc13cbf31c71.png">
Unsure if that is expected behavior or a residual bug.

Reply:
Great! Causing an actual error would be tricky (unless there's a way to do that I haven't heard about), but this approach gives me confidence in the change. Paypal is not supported in Elevate, so selecting it will deactivate the widget.